tests/getTask.test.ts

```diff
@@ -1,22 +1,23 @@
+import * as avs_pb from "../grpc_codegen/avs_pb";
 import { describe, beforeAll, test, expect } from "@jest/globals";
-import Client from "../dist/index.js";
+import Client from "../src";
 import dotenv from "dotenv";
 import path from "path";
 import { getAddress, generateSignature, requireEnvVar } from "./utils";
 
+import { sampleTask1, TEST_PRIVATE_KEY } from "./fixture";
```

Comment:
let’s give it a name with what it actually does, other than `sample`.

Reply:
updated